Dockerfile.node11-nsis

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:16.04
```

Comment:
Thinking out loud: Should we change this if it's working on 18 and instead solve why it's not working for the others?

Reply:
Funnily enough, I don't think it's working on 18, at least I wasn't able to get 18.x `nsis` images from this repo working in CI. Currently, if you ssh into a CI container it shows `16.04`. I think we had a mismatch of the image that was pushed versus the Dockerfiles tracked in this repo.

```
# node --version
v11.3.0

# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial
```

Reply:
You're right though, we should probably look into why changing the image causes the `apt-get` in the tests to fail.